### ISSUE TITLE
[Feature Request] Allow disabling of the reaction tooltip/popup thing

### ISSUE THREAD

**Issue by Hypfer** [telegramdesktop/tdesktop#17461]
### Is your feature request related to a problem?

After having updated to Beta 3.2.2, leaving the mouse hovering somewhere in a chat for too long lets the reaction popup appear.
This is unintentional because the mouse is just in a random position.

It's constantly happening to me that I'm reading some message just to be distracted by that random tooltip suddenly appearing ~600ms after I've stopped moving the mouse.

### Describe the solution you'd like

I'd like to be able to disable reactions on hover and instead react via the context menu.

### Describe alternatives you've considered

Allowing to disable reactions entirely would also be an option.

**Comment by TheStormN** [comment]
Yep, context menu is easily accessible on desktop platforms. Also it can be a little tiny emoji icon with shows on hover next to the 'reply' text button. After clicking it, the emoji popup should appear. That's just a suggestion to consider, but in general showing popup menus on desktop clients with hover timeout is not very intuitive.

**Comment by PeterNjeim** [comment]
Another big problem I have is that the emoji that pops up actually blocks text in certain instances. For example, it blocks the timestamp on other people's messages, it blocks the first few letters on your own messages, and it blocks the first reaction for your own already-reacted messages, meaning if you want to add that same reaction, if you don't do it fast enough it will add the wrong reaction. I would rather have a right click menu to add reactions, just like on the Android client. The fact you need to hover twice to pick the emoji you want is cumbersome. Or as the person above me has said, add an emoji icon on the message itself, and make it not obstruct any part of the message, possibly right next to the "admin" or "reply" text, similar to the one in the text field.

**Comment by john-preston** [comment]
@PeterNjeim It should be better in terms of blocking content in the 3.4.1 update.

**Comment by PeterNjeim** [comment]
> It should be better in terms of blocking content in the 3.4.1 update.

Thank you, works great

**Comment by PeterNjeim** [comment]
![image](https://user-images.githubusercontent.com/26418974/147834855-745229b5-f6a3-4d6e-87ec-3b1d6f3067ea.png)

Unfortunately this now covers the share button in channels lol, maybe move it up a bit in channels?

**Comment by PeterNjeim** [comment]
> ![image](https://user-images.githubusercontent.com/26418974/147834855-745229b5-f6a3-4d6e-87ec-3b1d6f3067ea.png)
> 
> Unfortunately this now covers the share button in channels lol, maybe move it up a bit in channels?

Thanks for fixing this, much appreciated
